### PR TITLE
feat(Video Game Mods): add presence

### PR DESCRIPTION
**Pull request by Kuriel23**
## Description 
Presence for Video Game Mods, a website for downloading mods for video games
Requested by issue #6663 

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>
image
</details>

### CODE REVIEW

File: websites/V/Video Game Mods/presence.ts [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+			],
+		},
+		urlpath = window.location.pathname.split("/");
+
+	switch (true) {
+		case !urlpath[1]:
+			presenceData.details = "Home";
+			break;
+		default:
+			if (urlpath[1].startsWith("members")) {
```
Comment (DarkVillager) [inline websites/V/Video Game Mods/presence.ts:25]: why?

you could have just done this?:
```suggestion
case urlpath[1].startsWith("members")
```

File: websites/V/Video Game Mods/presence.ts [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+				presenceData.details = `In ${
+					urlpath[3].length > 0 ? urlpath[3].replace("bp-", "") : "profile"
```
Comment (DarkVillager) [inline websites/V/Video Game Mods/presence.ts:27]: Or just get it with a querySelector?
```suggestion
					document.querySelector('[class="data"]')?.textContent
```

File: websites/V/Video Game Mods/presence.ts [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+				if (urlpath[2]) {
+					if (urlpath[2].startsWith("search")) {
+						presenceData.details = "Searching for:";
+						presenceData.state = decodeURI(urlpath[3]).replace("+", " ");
+						break;
+					} else {
+						presenceData.details = `In a discussion of ${
+							document.querySelector<HTMLHeadingElement>(
+								"a.bbp-breadcrumb-forum"
+							)?.innerText
```
Comment (DarkVillager) [inline websites/V/Video Game Mods/presence.ts:42]: Replace all innerText by textContent

File: websites/V/Video Game Mods/presence.ts [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+						}:`;
+						presenceData.state =
+							document.querySelector<HTMLHeadingElement>(
+								"h1.entry-title"
+							)?.innerText;
```
Comment (DarkVillager) [inline websites/V/Video Game Mods/presence.ts:47]: Replace all innerText by textContent

File: websites/V/Video Game Mods/presence.ts [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+							}`;
+							presenceData.state = document.querySelector<HTMLHeadingElement>(
+								"div.site-content h1"
+							)?.innerText;
+							presenceData.largeImageKey = document
+								.querySelector<HTMLImageElement>("div.modsmedia img")
+								?.getAttribute("src");
+							break;
+						} else {
+							presenceData.details = "Viewing Mods of";
```
Comment (DarkVillager) [inline websites/V/Video Game Mods/presence.ts:102]:
```suggestion
							presenceData.details = "Viewing mods of";
```

File: websites/V/Video Game Mods/presence.ts [hunk 6]
```diff
@@ -0,0 +1,159 @@
...
+							presenceData.state =
+								document.querySelector<HTMLHeadingElement>("h1")?.innerText;
+							break;
+						}
+					}
+					case "upload-mod":
+						presenceData.details = "Uploading Mod of";
```
Comment (DarkVillager) [inline websites/V/Video Game Mods/presence.ts:109]:
```suggestion
						presenceData.details = "Uploading mod of";
```

File: websites/V/Video Game Mods/presence.ts [hunk 7]
```diff
@@ -0,0 +1,159 @@
...
+						presenceData.state =
+							document.querySelector<HTMLHeadingElement>("h1")?.innerText;
+						break;
+					case "manage-mods":
+						presenceData.details = "Managing Mods of";
```
Comment (DarkVillager) [inline websites/V/Video Game Mods/presence.ts:114]:
```suggestion
						presenceData.details = "Managing mods of";
```

**Review by DarkVillager — CHANGES_REQUESTED** [review]

File: websites/V/Video Game Mods/metadata.json [hunk 8]
```diff
@@ -0,0 +1,28 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "354233941550694400",
+		"name": "Kuriel#1411"
```
Comment (Bas950) [inline websites/V/Video Game Mods/metadata.json:5]:
```suggestion
		"name": "Kuriel"
```

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/V/Video Game Mods/presence.ts [hunk 9]
```diff
@@ -0,0 +1,156 @@
...
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			details: "Other",
+			largeImageKey: "https://i.imgur.com/zvX0TaN.png",
+			startTimestamp: browsingTimestamp,
+			buttons: [
+				{
+					label: "View Page",
+					url: window.location.href,
```
Comment (theusaf) [inline websites/V/Video Game Mods/presence.ts:14]: In this repository, `document.location` is preferred over `window.location`
```suggestion
					url: document.location.href,
```

**Review by theusaf — CHANGES_REQUESTED** [review]

**Comment by DarkVillager** [comment]
Please resolves requested changes that you have changed.

**Review by Bas950 — APPROVED** [review]

**Review by EncryptedDev — APPROVED** [review]

**Pull request merged**